FFMediaToolkit/Encoding/Internal/OutputContainer.cs

```diff
@@ -16,12 +15,19 @@ internal unsafe class OutputContainer : Wrapper<AVFormatContext>
         private OutputContainer(AVFormatContext* formatContext)
             : base(formatContext)
         {
+            Video = new List<(OutputStream<VideoFrame>, VideoEncoderSettings)>();
+            Audio = new List<(OutputStream<AudioFrame>, AudioEncoderSettings)>();
         }
 
         /// <summary>
-        /// Gets the video stream.
+        /// Gets the video streams.
         /// </summary>
-        public OutputStream<VideoFrame> Video { get; private set; }
+        public List<(OutputStream<VideoFrame> stream, VideoEncoderSettings config)> Video { get; }
```

Comment:
maybe we should preserve the singular channel property, which I guess is pretty much 99% of cases, and this one should be a new property like AllVideoStreams  ?

Reply:
Yes, I agree, I'm working on making all the changes today and will be sure to include this.